components/AppNavbar.vue

```diff
@@ -63,6 +71,14 @@ const browserSupportExtension = extensionStore.url !== ''
     @on-brand-click="handleBrandClick"
   >
     <div class="w-full flex items-center justify-end" slot="desktop">
+      <lukso-button
+        v-if="status.isConnected"
```

Comment:
I think we can remove `v-if` here, this link should be always visible.

Reply:
yes make sense. 👍🏼